pkg/reconciler/namespace/namespace.go

```diff
@@ -190,11 +196,11 @@ func (c *Controller) assignCluster(ctx context.Context, ns *corev1.Namespace) er
 
 	oldClusterName := ns.Labels[clusterLabel]
 	if oldClusterName != newClusterName {
-		klog.Infof("Patching to update cluster assignment for namespace %q|%q: %q -> %q", ns.ClusterName, ns.Name, oldClusterName, newClusterName)
-		if _, err := c.kubeClient.Cluster(ns.ClusterName).CoreV1().Namespaces().Patch(ctx, ns.Name, types.MergePatchType, clusterLabelPatchBytes(newClusterName), metav1.PatchOptions{}); err != nil {
+		klog.Infof("Patching to update scheduled cluster for namespace %q|%q: %q -> %q", ns.ClusterName, ns.Name, oldClusterName, newClusterName)
```

Comment:
Why do you propose to update 'cluster assignment' to 'scheduled cluster' everywhere? A resource is scheduled to a cluster. A resource has a cluster assignment/a cluster assigned to it. A resource doesn't have a scheduled cluster.

Reply:
Agreed, I've reverted the change. I think that was consistency gone wrong :)